src/views/AccountingOverview/internal/TransactionsToReview.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { Badge } from '../../../components/Badge'
+import { BadgeSize, BadgeVariant } from '../../../components/Badge/Badge'
+import { BadgeLoader } from '../../../components/BadgeLoader'
+import { IconButton } from '../../../components/Button'
+import { ProfitAndLoss } from '../../../components/ProfitAndLoss'
+import { Text, TextSize } from '../../../components/Typography'
+import { VStack } from '../../../components/ui/Stack'
+import { useProfitAndLossLTM } from '../../../hooks/useProfitAndLoss/useProfitAndLossLTM'
+import BellIcon from '../../../icons/Bell'
+import CheckIcon from '../../../icons/Check'
+import ChevronRight from '../../../icons/ChevronRight'
+import RefreshCcw from '../../../icons/RefreshCcw'
+import type { Variants } from '../../../utils/styleUtils/sizeVariants'
+import { getMonth, getYear, startOfMonth } from 'date-fns'
+
+const CLASS_NAME = 'Layer__TransactionsToReview'
+
+type TransactionsToReviewProps = {
+  onClick?: () => void
+  usePnlDateRange?: boolean
+  tagFilter?: {
+    key: string
+    values: string[]
+  }
+  variants?: Variants
+}
+
+export function TransactionsToReview({
```

Comment:
This file was renamed, so it appears as completely new.

The goal here was to change as little as possible.